pkg/template/event.go

```diff
@@ -27,8 +27,13 @@ import (
 )
 
 // bodyPathVarRegex determines valid body path variables
-var bodyPathVarRegex = regexp.MustCompile(`\$\(body(.[0-9A-Za-z_-]+)*\)`)
-var headerVarRegex = regexp.MustCompile(`\$\(header(.[0-9A-Za-z_-]+)?\)`)
+// The body regular expression allows for a subset of GJSON syntax, the mininum
+// required to navigate through dictionaries, query arrays and support
+// namespaced label names e.g. tekton.dev/eventlistener
+var bodyPathVarRegex = regexp.MustCompile(`\$\(body(\.[[:alnum:]/_\-\.\\]+|\.#\([[:alnum:]=<>%!"\*_-]+\)#??)*\)`)
+
+// The headers regular expression allows for simple
```

Comment:
I think that this comment got cut off or something?

Reply:
Good point, thanks!